cmd/cloud/deployment_objects.go

```diff
@@ -0,0 +1,645 @@
+	}
+
+	if connType == "" {
+		return errors.New("a connection type is needed to create a connection. Please use the '--conn-type' flag to specify a connection type")
+	}
+
+	return deployment.ConnectionCreate(airlfowURL, connID, connType, description, host, login, password, schema, extra, port, airflowAPIClient, out)
+}
+
+func deploymentConnectionUpdate(cmd *cobra.Command, out io.Writer) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid workspace")
+	}
+
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	requestedField = requestString
+	// get Airflow URl
+	value, err := inspect.ReturnSpecifiedValue(ws, deploymentName, deploymentID, astroClient, requestedField)
+	if err != nil {
+		return errors.Wrap(err, "failed to find a deployments airflow webserver URL")
+	}
+
+	airlfowURL := fmt.Sprintf("%v", value)
+
+	// check connID and connType
+	if connID == "" {
+		return errors.New("a connection ID is needed to create a connection. Please use the '--conn-id' flag to specify a connection ID")
+	}
+
+	return deployment.ConnectionUpdate(airlfowURL, connID, connType, description, host, login, password, schema, extra, port, airflowAPIClient, out)
+}
+
+//nolint:dupl
+func deploymentConnectionCopy(cmd *cobra.Command, out io.Writer) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid workspace")
+	}
+
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	requestedField = requestString
+	// get To Airflow URls
+	if fromDeploymentName == "" && fromDeploymentID == "" {
+		fmt.Println("Which deployment should connections be copied from?")
+	}
+	fromValue, err := inspect.ReturnSpecifiedValue(ws, fromDeploymentName, fromDeploymentID, astroClient, requestedField)
+	if err != nil {
+		return errors.Wrap(err, "failed to find the source deployments airflow webserver URL")
+	}
+
+	fromAirlfowURL := fmt.Sprintf("%v", fromValue)
+	if toDeploymentName == "" && toDeploymentID == "" {
+		fmt.Println("Which deployment should connections be pasted to?")
+	}
+	value, err := inspect.ReturnSpecifiedValue(ws, toDeploymentName, toDeploymentID, astroClient, requestedField)
+	if err != nil {
+		return errors.Wrap(err, "failed to find a deployments airflow webserver URL")
+	}
+
+	toAirlfowURL := fmt.Sprintf("%v", value)
+
+	return deployment.CopyConnection(fromAirlfowURL, toAirlfowURL, airflowAPIClient, out)
+}
+
+func deploymentAirflowVariableList(cmd *cobra.Command, out io.Writer) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid workspace")
+	}
+
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	requestedField = requestString
+	// get Airflow URl
+	value, err := inspect.ReturnSpecifiedValue(ws, deploymentName, deploymentID, astroClient, requestedField)
```

Comment:
Please update these details for the rest of the error messages:

- Capitalize Deployment, Workspace, and Airflow in all uses.
- Don't use possessive language for Deployments (ie. do not write "Deployment's)
- Which deployment should X be pasted to? => Which Deployment should receive X?